lcoe_calculator/tech_processors.py

```diff
@@ -21,13 +21,22 @@
 
 
 class OffShoreWindProc(TechProcessor):
+    #Abstract class, sheet name is not defined. See Fixed and Floating OSW proc
```

Comment:
Suggest converting this to a doc string `""" ... """` for better intellisense support, etc.